app/src/backlight.c

```diff
@@ -0,0 +1,237 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <device.h>
+#include <init.h>
+#include <kernel.h>
+
+#include <drivers/led.h>
+#include <logging/log.h>
+#include <settings/settings.h>
+
+#include <zmk/backlight.h>
```

Comment:
I don't understand, `""` makes sense for a header, but in a `.c` file there is no local file to include. Also I don't see any files that use the `""` syntax.

Reply:
After looking at the C standard, what I said is not technically accurate. Search order for both quotes and brackets is implementation defined. https://stackoverflow.com/a/77092

However, if we go by what GCC says, bracket should be used for system header files, while quotes should be "used for header files of your own program". https://gcc.gnu.org/onlinedocs/cpp/Include-Syntax.html

Reply:
I think if you want to use `""` instead of `<>` you should refactor the whole project and possibly in a separate PR.
Using different styles in different files is bad and I want to be consistent with the rest of the project.

Reply:
Okay, looking at the rest of the codebase, `""` is used, but only in one or two spots. This is something that should be discussed separately from this PR.